ship/src/main/scala/ch/epfl/bluebrain/nexus/ship/files/FileCopier.scala

```diff
@@ -14,7 +14,7 @@ import scala.concurrent.duration.DurationInt
 
 trait FileCopier {
 
-  def copyFile(project: ProjectRef, attributes: FileAttributes): IO[Uri.Path]
+  def copyFile(project: ProjectRef, attributes: FileAttributes): IO[Option[Uri.Path]]
```

Comment:
It's not very intuitive what the option represents. It would be clearer with an ADT

Reply:
You are right, I will handle it in my next PR which also implies changes on this part